blueocean-dashboard/src/main/js/creation/git/GitDefaultOption.js

```diff
@@ -15,9 +15,11 @@ export default function GitDefaultOption(props) {
         }
     }
 
+    const styles = `git-creation monochrome ${props.isSelected ? 'active' : ''}`;
```

Comment:
minor nit : I'd name this `classNames` as we use that convention throughout, and at a glance most folks might assume that `styles` would be a React style object rather than list of classes.